src/main/java/frc/robot/util/Bot2020DrivePorts.java

```diff
@@ -71,17 +64,17 @@ public double getWheelDiameterInInches() {
     @Override
     public double getAutoDistanceKP() {
         // TODO Auto-generated method stub
-        return autoDistanceKP;
+        return autoDistanceBot2020PID.kp;
     }
     @Override
     public double getAutoDistanceKI() {
         // TODO Auto-generated method stub
-        return autoDistanceKI;
+        return autoDistanceBot2020PID.ki;
     }
     @Override
     public double getAutoDistanceKD() {
         // TODO Auto-generated method stub
-        return autoDistanceKD;
+        return autoDistanceBot2020PID.kd;
```

Comment:
Instead of just passing the values from the pid config though the bot drive ports. It would be cleaner to change the method definition inside of hte "IDrivePorts" interface to get the whole PIDConfig object vice the each coefficient. 

IE
```java
public PIDConfig getAutoDistancePID();
```